src/main/java/bookface/logic/commands/AddUserCommand.java

```diff
@@ -1,47 +1,49 @@
-package seedu.address.logic.commands;
+package bookface.logic.commands;
 
+import static bookface.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static bookface.logic.parser.CliSyntax.PREFIX_NAME;
+import static bookface.logic.parser.CliSyntax.PREFIX_PHONE;
+import static bookface.logic.parser.CliSyntax.PREFIX_TAG;
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 
-import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.Model;
-import seedu.address.model.person.Person;
+import bookface.logic.commands.exceptions.CommandException;
+import bookface.model.Model;
+import bookface.model.person.Person;
 
 /**
  * Adds a person to the address book.
  */
-public class AddCommand extends Command {
+public class AddUserCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String COMMAND_WORD_USER = " user";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + COMMAND_WORD_USER
+            + ": Adds a user to the address book. "
```

Comment:
minor aesthetic issue in line 23, could change "to the address book" to "to the user list"